### DISCUSSION TITLE
Hide short Videos?

### DISCUSSION THREAD

**Discussion by dostnmatr** [libre-tube/LibreTube#3384] [General]
Hi, I would really like to filter out all videos shorter than five minutes by default since I find them distracting and really do not want to see any of them. Is that possible with the current version to hide all videos shorter than x minutes?
There was an Issue submitted about that (#1521) and closed because "not planned". In the meantime, shorts have become much more frequent and Youtube is actively pushing for more short content. With my bookmarked channels, short videos are mostly self-promotion or just ads or opinion without reasoning etc. They are distracting and I feel that these videos waste my time. That's why I would like to hide them.
Is this already possible or or would you take this feature under consideration again?
Thanks!

**Comment by erkkimon** [comment]
+1 because shorts are basically waste of screen estate, at least if they are not organized like on YouTube. And there is annoyingly huge amount of shorts floating around YouTube nowadays.

**Comment by luckkmaxx** [comment]
Right now you can use the filter only for your feed, wich can hide shorts (#shorts, not "short" videos).
It is in the up-left corner.

**Comment by robertosw** [comment]
I actively searched for a YouTube app thats not showing shorts in the recommendations and subscriptions and I am really happy with this app for having a filter to hide shorts.
Please never add a "shorts" tab like YouTubes offical app has. 

Love the app in its current state 👌
